examples/src/legacy/multiple_kms_cmk_regions.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+Example showing basic encryption and decryption of a value already in memory
+using multiple KMS CMKs in multiple regions.
+"""
+import aws_encryption_sdk
+from aws_encryption_sdk.key_providers.kms import KMSMasterKey, KMSMasterKeyProvider
+
+
+def run(aws_kms_generator_cmk, aws_kms_child_cmks, source_plaintext, botocore_session=None):
+    """Encrypts and then decrypts a string under multiple KMS customer master keys (CMKs) in multiple regions.
+
+    :param str aws_kms_generator_cmk: Amazon Resource Name (ARN) of the primary KMS CMK
+    :param List[str] aws_kms_child_cmks: Additional Amazon Resource Names (ARNs) of secondary KMS CMKs
+    :param bytes source_plaintext: Data to encrypt
+    :param botocore_session: existing botocore session instance
+    :type botocore_session: botocore.session.Session
+    """
+    child_cmk = aws_kms_child_cmks[0]
+    # Check that these keys are in different regions
+    assert not aws_kms_generator_cmk.split(":")[3] == child_cmk.split(":")[3]
+
+    kwargs = dict(key_ids=[aws_kms_generator_cmk, child_cmk])
+
+    if botocore_session is not None:
+        kwargs["botocore_session"] = botocore_session
+
+    # Create master key provider using the ARNs of the keys and the session (botocore_session)
+    kms_key_provider = KMSMasterKeyProvider(**kwargs)
+
+    # Encrypt the plaintext using the AWS Encryption SDK. It returns the encrypted message and the header
+    ciphertext, encrypted_message_header = aws_encryption_sdk.encrypt(
+        key_provider=kms_key_provider, source=source_plaintext
+    )
+
+    # Check that both key ARNs are in the message headers
+    assert len(encrypted_message_header.encrypted_data_keys) == 2
```

Comment:
Do we need to verify these are actually ARNs?

Reply:
IMO that doesn't really add much to the example, but I'm willing to be convinced otherwise.

Reply:
From a correctness standpoint, I think it would be good to check them (or at least note that "in production, we would want to further verify the ARNs are valid" or something similar in case someone copy/pastes part of the example and forgets to check.

Reply:
I think that the right thing there will be to include an example of interacting with the keyring trace and using that to verify that the CMKs that you expected to encrypt did actually encrypt and that the one you expected to generate did actually generate.